src/pg_to_evalscript/javascript_processes/gt.js

```diff
@@ -0,0 +1,31 @@
+function gt(arguments) {
+  const { x, y } = arguments;
+
+  if (x === undefined) {
+    throw Error("Process gt requires argument x.");
+  }
+
+  if (y === undefined) {
+    throw Error("Process gt requires argument y.");
+  }
+
+  if (x === null || y === null) {
+    return null;
+  }
+
+  if (
+    (typeof x !== "number" && typeof x !== "string") ||
+    (typeof y !== "number" && typeof y !== "string")
+  ) {
+    return false;
```

Comment:
Specifically

> If any operand is not a number or **temporal string** (date, time or date-time), the process returns false.

So I assume `gt({"x":"b", "y": "a"})` should return `false`.

Similar for `lt`.

Reply:
was added, but we will have to think of something else if there will be more interdependent